src/link_renderer.ts

```diff
@@ -120,7 +119,7 @@ export class LinksRenderer {
       var $combi = $group.selectAll('g').data(combinations);
       $combi.enter().append('g');
       $combi.exit().remove();
-      $combi.attr('data-id', identity);
+      $combi.attr('data-id', String);
```

Comment:
poor man toString ... converts it to a string. You see this pattern now and then in the d3 context instead of `(d) => d` if the result should be a string.